src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    public static final Map<String, Integer> fruitStorage = new HashMap<>();
```

Comment:
Following the encapsulation principle, the access to `fruitStorage` should be restricted. It's better to make it private and provide access through methods.